src/internal/connector/onedrive/collection.go

```diff
@@ -42,6 +42,9 @@ const (
 	MetaFileSuffix    = ".meta"
 	DirMetaFileSuffix = ".dirmeta"
 	DataFileSuffix    = ".data"
+
+	// Used to compare in case of OneNote files
+	Size2GB = 2 * 1024 * 1024 * 1024
```

Comment:
nit: a slightly more descriptive name can make reading this var in code easier
```suggestion
	MaxOneNoteFileSize = 2 * 1024 * 1024 * 1024
```